monkey/monkey_island/cc/main.py

```diff
@@ -25,9 +26,24 @@
 from monkey_island.cc.environment.environment import env
 from monkey_island.cc.database import is_db_server_up, get_db_version
 from monkey_island.cc.resources.monkey_download import MonkeyDownload
+from monkey_island.cc.bootloader_server import BootloaderHttpServer
 
 
 def main():
+
+    logger.info("Starting bootloader server")
+    mongo_url = os.environ.get('MONGO_URL', env.get_mongo_url())
+    bootloader_server_thread = Thread(target=BootloaderHttpServer(mongo_url).serve_forever, daemon=True)
+    # island_server_thread = Thread(target=start_island_server)
+
+    bootloader_server_thread.start()
+    #island_server_thread.start()
+    start_island_server()
+    bootloader_server_thread.join()
```

Comment:
Extract all this to function

Reply:
What do you mean? It's already the only lines lines in this function.